Jenkinsfile

```diff
@@ -129,17 +129,14 @@ echo ${libs} | sed -e 's/,/ /g' | xargs md5sum
 // Python 2
 def python2_ut(docker_type) {
   timeout(time: max_time, unit: 'MINUTES') {
-    sh "${docker_run} ${docker_type} --dockerbinary docker find . -name '*.pyc' -type f -delete"
-    sh "${docker_run} ${docker_type} --dockerbinary docker PYTHONPATH=./python/ nosetests-2.7 --with-timer --verbose tests/python/unittest"
-    sh "${docker_run} ${docker_type} --dockerbinary docker PYTHONPATH=./python/ nosetests-2.7 --with-timer --verbose tests/python/train"
+    sh "ci/build.py --build -p ${docker_type} /work/runtime_functions.sh unittest_ubuntu_python2_cpu"
```

Comment:
I like it, it's shorter, less noisy, reproducible locally....

Reply:
we can also call build.py with -bp why one option long and the other short? otherwise for consistency both long? --build --platform x